python/feathub/feature_views/transforms/python_transform.py

```diff
@@ -18,16 +18,20 @@
 from feathub.feature_views.transforms.transformation import Transformation
 
 
-# TODO: support python udf.
 class PythonUdfTransform(Transformation):
     """
     Derives feature values by applying a Python UDF on one row of the parent table at a
     time.
     """
 
-    def __init__(self, callable: Callable[[pd.DataFrame], Any]) -> None:
+    def __init__(self, callable_: Callable[[pd.Series], Any]) -> None:
```

Comment:
This is named `callable_` because `callable` is a built-in name.

How about re-naming it `udf`?